lua/lvim/lsp/config.lua

```diff
@@ -118,6 +118,8 @@ return {
     insert_mode = {},
     visual_mode = {},
   },
+  ---@usage list of settings of nvim-lsp-installer
+  installer = {},
```

Comment:
```suggestion
  installer = {
    -- use the default nvim_data_dir, since the server binaries are independent
    install_root_dir = utils.join_paths(vim.call("stdpath", "data"), "lsp_servers"),
  },
```

Reply:
As i see in the default setting of `nvim-lsp-installer`, it sets `install_root_dir` as the same as us. Can i completely remove this setting?

https://github.com/williamboman/nvim-lsp-installer/blob/5904749c2b7fb9d70967a9dc3599107e3d07c6d1/lua/nvim-lsp-installer/settings.lua#L53-L54

Reply:
I have added some default settings.